sdk/tests/conformance/offscreencanvas/context-attribute-preserve-drawing-buffer.html

```diff
@@ -41,33 +41,83 @@
 "use strict";
 description("This test checks whether OffscreenCanvas webgl context honors the preserveDrawingBuffer flag.");
 var wtu = WebGLTestUtils;
+var pixels = new Uint8Array(4);
 
-const nextFrame = async () => new Promise(r => requestAnimationFrame(r));
+function checkPixels(color) {
+  return (color[0] === pixels[0]) &&
+    (color[1] === pixels[1]) &&
+    (color[2] === pixels[2]) &&
+    (color[3] === pixels[3]);
+}
+
+const nextFrame = () => new Promise(r => requestAnimationFrame(r));
 
-async function getPixelsFromOffscreenWebgl(preserveFlag, color, msg) {
+async function getPixelsFromOffscreenWebglPreserveFlagTrue(color, msg) {
   var canvas = document.createElement("canvas");
   var offscreenCanvas = transferredOffscreenCanvasCreation(canvas, 10, 10);
-  var gl = offscreenCanvas.getContext("webgl", {preserveDrawingBuffer: preserveFlag});
+  var gl = offscreenCanvas.getContext("webgl", {preserveDrawingBuffer: true});
 
   // Draw some color on gl
   gl.clearColor(1, 0, 1, 1);
   gl.clear(gl.COLOR_BUFFER_BIT);
 
-  await nextFrame();
-  var pixels = new Uint8Array(4);
-  wtu.checkCanvas(gl, color, msg);
+  var t, t0 = await nextFrame();
```

Comment:
I prefer to avoid declaring multiple variables on the same line when one has an initializer, as it can be confusing. Maybe put t0 on its own line and also declare it const instead of var. While we're rewriting this code, maybe also change var to let (or const as appropriate) throughout the file.